src/main/java/com/kodilla/ecommercee/entity/User.java

```diff
@@ -0,0 +1,35 @@
+package com.kodilla.ecommercee.entity;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+
+@Entity
+@Getter
+@AllArgsConstructor
+@NoArgsConstructor
+public class User {
+
+    @Id
+    @GeneratedValue
+    private Long userId;
+
+    private String name;
+
+    private String password;
+
+    private boolean blocked;
+
+    private String address;
+
+    public User(String name, String password, boolean blocked, String address){
```

Comment:
Wygląda jakby te linijki pod spodem były nie sformatowane. Odpal ctrl + alt + L w intellij żeby zrobić to automatycznie